cvmfs/cvmfs_config

```diff
@@ -1231,9 +1235,13 @@ add_user_to_group_fuse() {
 check_auto_mounter() {
   case $sys_arch in
     Linux )
-      $service autofs status > /dev/null 2>&1
-      return $? ;;
-
+        if [[ $service == *systemctl ]]; then
```

Comment:
I see, alright then let's leave this spot as is.  We'll probably later make another round of modifications to hide the specifics of service management.